components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Microsoft.Azure.IIoT.OpcUa.Protocol.csproj

```diff
@@ -22,11 +22,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.15" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Ini" Version="3.1.15" />
-    <PackageReference Include="System.Net.Security" Version="4.3.2" />
-    <PackageReference Include="System.Net.WebSockets.Client" Version="4.3.2" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="5.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Ini" Version="6.0.0" />
```

Comment:
The rest of the projects are using `3.1.15` version of `Microsoft.Extensions.Configuration.*` packages. I am wondering if we should upgrade all of them as well or leave them be. Is the `6.0.0` version required here only because of the version of `Microsoft.Extensions.Logging` package ?

Reply:
the latest Nuget version allows to use the 3.1 version of the extension packages